multicluster/controllers/multicluster/common/helper.go

```diff
@@ -52,3 +54,21 @@ func RemoveString(slice []string, s string) (result []string) {
 	}
 	return
 }
+
+// FilterSubsets keeps IPs only and remove others which are unnecessary information for other member cluster.
+func FilterSubsets(subsets []corev1.EndpointSubset) []corev1.EndpointSubset {
+	newSubsets := []corev1.EndpointSubset{}
```

Comment:
Nit: we can make a slice with the same size of len(subsets).

Reply:
the final might not be equal when there is no available Addresses but only NotReadyAddress, we only care about those addresses which are ready.

Reply:
Sounds good.